content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+manufacturers, so it is likely new models will need to be added to the database to correctly
+configure a fleet of vehicles.
+
+Preconfigured Models
+--------------------
+
+The following models are pre-loaded in Odoo and do not need to be added to the database:
+
++-------+--------------+-----------+---------------+
+| AUDI  | BMW          | Mercedes  |     Opel      |
++=======+==============+===========+===============+
+| A1    | Serie 1      | Class A   | Agilia        |
++-------+--------------+-----------+---------------+
+| A3    | Serie 3      | Class B   | Ampera        |
++-------+--------------+-----------+---------------+
+| A4    | Serie 5      | Class C   | Antara        |
++-------+--------------+-----------+---------------+
+| A5    | Serie 6      | Class CL  | Astra         |
++-------+--------------+-----------+---------------+
+| A6    | Serie 7      | Class CLS | AstraGTC      |
++-------+--------------+-----------+---------------+
+| A7    | Serie Hybrid | Class E   | Combo Tour    |
++-------+--------------+-----------+---------------+
+| A8    | Serie M      | Class GL  | Corsa         |
++-------+--------------+-----------+---------------+
+| Q3    | Serie X      | Class GLK | Insignia      |
++-------+--------------+-----------+---------------+
+| Q5    | Serie Z4     | Class M   | Meriva        |
++-------+--------------+-----------+---------------+
+| TT    |              | Class R   | Mokka         |
++-------+--------------+-----------+---------------+
+|       |              | Class S   | Zafira        |
++-------+--------------+-----------+---------------+
+|       |              | Class SLK | Zafira Tourer |
++-------+--------------+-----------+---------------+
+|       |              | Class SLS |               |
++-------+--------------+-----------+---------------+
+
+.. _configuration/add-model:
+
+Add a new model
+---------------
+
+New vehicle models can easily be added to the database. To add a new model, navigate to
+:menuselection:`Fleet app --> Configuration --> Vehicle Models`. Click :guilabel:`Create` and a
+vehicle model form will load. Enter the following information on the form, then click
+:guilabel:`Save`. Be advised, some fields are specific to Belgian based companies, so not all fields
+or sections may be visible depending on the location of the company.
+
+- :guilabel:`Model name`: Enter the model name in the field.
+- :guilabel:`Manufacturer`: Select the manufacturer from the drop-down menu. If the manufacturer is
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  & Edit`.
+- :guilabel:`Vehicle Type`: Select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application. Adding additional vehicle types is not possible as it
+  will affect payroll.
```

Comment:
This makes me want to know why this is? Why is it integrated into payroll?

Reply:
I added some more text here- vehicles are part of compensation/benefits/etc, so that's why there are parts of fleet that are firmly connected to Payroll.